__tests__/components/sections/homepage/OpenSource.test.tsx

```diff
@@ -0,0 +1,14 @@
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import { renderWithRedux } from "../../../../__mocks__/renderWithRedux";
+import OpenSource from "@/components/sections/homepage/OpenSource";
+
+describe("OpenSource", () => {
+  it("should render Open Source", () => {
+    renderWithRedux(<OpenSource />);
+    expect(screen.getByTestId("openSourceId")).toBeInTheDocument();
+    expect(screen.getByRole("link")).toBeInTheDocument();
+    expect(screen.getByRole("link").hasAttribute("href")).toBeTruthy();
```

Comment:
avoid repetition by using link element variable 
```suggestion
     const link = screen.getByRole("link");
    expect(link).toBeInTheDocument();
    expect(link).toHaveAttribute("href", "https://github.com/dacadeorg/dacade-frontend-app");
```